zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/BackupActuator.java

```diff
@@ -112,6 +112,10 @@ static BackupActuator of(final String endpoint) {
   @Headers({"Content-Type: application/json", "Accept: application/json"})
   List<BackupInfo> list();
 
+  @RequestLine("GET /{prefix}")
+  @Headers({"Content-Type: application/json", "Accept: application/json"})
+  List<BackupInfo> list(@Param final String prefix);
```

Comment:
❓ The other list will always do with wildcard right?

Reply:
Exactly :+1: That's how the API works now.